conans/client/conanfile/configure.py

```diff
@@ -9,6 +9,8 @@ def run_configure_method(conanfile, down_options, profile_options, ref):
     if hasattr(conanfile, "config_options"):
         with conanfile_exception_formatter(conanfile, "config_options"):
             conanfile.config_options()
+    else:
+        conanfile._conan_config_options_default()
```

Comment:
The protected method has the advantage when using an IDE with autocomplete, that you probably will not use the method in your recipe. But don't see problem by using public method here, since it's just a tool and should not violate the design access. Anyway, python allows calling protected methods just like public, it will complain, but will not break.

Reply:
Yes, I think it would be better to not make it a private method of ``ConanFile``, but directly call the ``handle`` method instead.